lib/config/config-validator.js

```diff
@@ -97,7 +101,10 @@ function validateRuleOptions(id, options, source) {
 
         if (!validSeverity) {
             message.push(
-                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed \"", severity, "\").\n");
+                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed `",
+                util.inspect(severity).replace(/'/g, "\"").replace(/\n/g, ""),
```

Comment:
Oops, sorry. What I meant is that we use single quotes in error messages throughout ESLint already ("unexoected value 'true'"), so let's just stick with what we are already doing.